sherpa/plot/backends.py

```diff
@@ -0,0 +1,538 @@
+#
+#  Copyright (C) 2007, 2015, 2020, 2021  Smithsonian Astrophysical Observatory
+#
+#
+#  This program is free software; you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation; either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License along
+#  with this program; if not, write to the Free Software Foundation, Inc.,
+#  51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+import logging
+
+from sherpa.utils import get_keyword_defaults
+from sherpa.utils import formatting
+
+
+__all__ = ('BasicBackend')
```

Comment:
I know this gets changed in a later commit, but technically I believe this should be 

```suggested
__all__ = ('BasicBackend', )
```